README.md

```diff
@@ -1,20 +1,23 @@
 # Wrangler
-<img src="https://cdap-users.herokuapp.com/assets/cm-available.svg"/> <img src="https://cdap-users.herokuapp.com/assets/cdap-transform.svg"/>
-<a href="https://cdap-users.herokuapp.com/"><img alt="Join CDAP community" src="https://cdap-users.herokuapp.com/badge.svg?t=wrangler"/></a> [![Build Status](https://travis-ci.org/hydrator/wrangler.svg?branch=develop)](https://travis-ci.org/hydrator/wrangler)
-<a href="https://scan.coverity.com/projects/hydrator-wrangler-transform">
-  <img alt="Coverity Scan Build Status"
-       src="https://scan.coverity.com/projects/11434/badge.svg"/>
-</a>
+
+![cm-available](https://cdap-users.herokuapp.com/assets/cm-available.svg)
+![cdap-transform](https://cdap-users.herokuapp.com/assets/cdap-transform.svg)
+[![Join CDAP community](https://cdap-users.herokuapp.com/badge.svg?t=wrangler)](https://cdap-users.herokuapp.com)
+[![Build Status](https://travis-ci.org/hydrator/wrangler.svg?branch=develop)](https://travis-ci.org/hydrator/wrangler)
+[![Coverity Scan Build Status](https://scan.coverity.com/projects/11434/badge.svg)](https://scan.coverity.com/projects/hydrator-wrangler-transform)
 [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-Collection of libraries, pipeline plugin and CDAP service for performing data cleansing, transformation and filtering using a set of instructions. Instructions to manipulate data are either generated using an interative visual tool or could be manually entered.
+A collection of libraries, a pipeline plugin, and a CDAP service for performing data
+cleansing, transformation, and filtering using a set of data manipulation instructions.
```

Comment:
do you want to call them "directives" instead of "data manipulation instructions"? Or maybe put directives in () afterwards.